content/en/docs/concepts/security/service-accounts.md

```diff
@@ -0,0 +1,232 @@
+  situations such as the following:
+  *  Providing read-only access to sensitive information stored in Secrets.
+  *  Granting [cross-namespace access](#cross-namespace), such as allowing a
+     Pod in namespace `example` to read, list, and watch for Lease objects in
+     the `kube-node-lease` namespace.
+* Your Pods need to communicate with an external service. For example, a
+  workload Pod requires an identity for a commercially available cloud API,
+  and the commercial provider allows configuring a suitable trust relationship.
+* [Authenticating to a private image registry using an `imagePullSecret`](/docs/tasks/configure-pod-container/configure-service-account/#add-imagepullsecrets-to-a-service-account).
+* An external service needs to communicate with the Kubernetes API server. For
+  example, authenticating to the cluster as part of a CI/CD pipeline.
+* You use third-party security software in your cluster that relies on the
+  ServiceAccount identity of different Pods to group those Pods into different
+  contexts.
+
+
+## How to use service accounts {#how-to-use}
+
+To use a Kubernetes service account, you do the following:
+
+1. Create a ServiceAccount object using a Kubernetes
+   client like `kubectl` or a manifest that defines the object.
+1. Grant permissions to the ServiceAccount object using an authorization
+   mechanism such as
+   [RBAC](https://kubernetes.io/docs/reference/access-authn-authz/rbac/).
+1. Assign the ServiceAccount object to Pods during Pod creation.
+
+For instructions, refer to
+[Configure Service Accounts for Pods](/docs/tasks/configure-pod-container/configure-service-account/).
+
+### Grant permissions to a ServiceAccount {#grant-permissions}
+
+You can use the built-in Kubernetes
+[role-based access control (RBAC)](/docs/reference/access-authn-authz/rbac/)
+mechanism to grant the minimum permissions required by each service account.
+You create a *role*, which grants access, and then *bind* the role to your
+ServiceAccount. RBAC lets you define a minimum set of permissions so that the
+service account permissions follow the principle of least privilege. Pods that
+use that service account don't get more permissions than are required to
+function correctly.
+
+#### Cross-namespace access using a ServiceAccount {#cross-namespace}
+
+You can use RBAC to allow service accounts in one namespace to perform actions
+on resources in a different namespace in the cluster. For example, consider a
+scenario where you have a service account and Pod in the `dev` namespace and
+you want your Pod to see Jobs running in the `maintenance` namespace. You could
+create a Role object that grants permissions to list Job objects. Then,
+you'd create a RoleBinding object in the `maintenance` namespace to bind the
+Role to the ServiceAccount object. Now, Pods in the `dev` namespace can list
+Job objects in the `maintenance` namespace using that service account.
+
```

Comment:
:thought_balloon: 

_Doesn't_ block this PR, but you can also allow one ServiceAccount to impersonate another, so there are two options here.

Reply:
Impersonation is always possible, the current documented process appears cleaner to me since you're granting the access explicitly, rather than inheriting ALL the access of the other SA.

Reply:
Let's be prescriptive and document the approach that's "better" from us